src/View/Helper/SchemaHelper.php

```diff
@@ -234,4 +238,46 @@ function ($value) {
 
         return compact('type', 'value');
     }
+
+    /**
+     * Provides list of translatable fields per schema properties
+     *
+     * @param array $properties The array of schema properties
+     * @return array
+     */
+    public function translatableFields(array $properties) : array
+    {
+        if (empty($properties)) {
+            return [];
+        }
+
+        $fields = [];
+        foreach ($properties as $name => $property) {
+            if (!empty($property['oneOf'])) {
```

Comment:
[minor] probably another method to check single `$property` match will make this more readable
[it's a matter of taste... you can leave it like this]

Reply:
I would leave `code beautifying` to a "refactor schema helper" issue ;)

Reply:
I agree